### PR TITLE
bugtool: fetch gops pprof heap

### PR DESCRIPTION
**Pull request by tixxdz**

### CODE REVIEW

File: pkg/bugtool/bugtool.go [hunk 1]
```diff
@@ -83,6 +84,14 @@ func doSaveInitInfo(fname string, info *InitInfo) error {
 		logger.GetLogger().WithField("bpftool", info.BpfToolPath).Info("Successfully detected bpftool path")
 	}
 
+	gops, err := exec.LookPath("gops")
```
Comment (kkourt) [inline pkg/bugtool/bugtool.go:89]: Why do we need to do this on the agent?

Comment (tixxdz) [reply pkg/bugtool/bugtool.go:89]: cause IIRC the agent has the path of installation in systemd , like bpftool so it finds the path for later

Comment (tixxdz) [reply pkg/bugtool/bugtool.go:89]: So, we get the path in the agent we save it into the tetragon json inside /run/tetragon/, and later tetra tries to auto read it allowing users to get dump without specifying the search path or the full gops path.

File: pkg/bugtool/bugtool.go [hunk 2]
```diff
@@ -498,11 +511,26 @@ func (s *bugtoolInfo) addBpftoolInfo(tarWriter *tar.Writer) {
...
+	_, err := os.Stat(s.info.GopsPath)
+	if err != nil {
+		s.multiLog.WithField("gops-address", s.info.GopsAddr).WithError(err).Warn("Failed to locate gops, please install it.")
+		return
+	}
+
+	s.multiLog.WithField("gops-address", s.info.GopsAddr).WithField("gops-path", s.info.GopsPath).Info("Dumping gops information")
+
+	s.execCmd(tarWriter, "gops.stack", s.info.GopsPath, "stack", s.info.GopsAddr)
+	s.execCmd(tarWriter, "gpos.stats", s.info.GopsPath, "stats", s.info.GopsAddr)
```
Comment (kkourt) [inline pkg/bugtool/bugtool.go:561]: nit: could you also fix the typo here please? gpos -> gops

**Review by kkourt — APPROVED** [review]
LGTM, thanks!

File: pkg/bugtool/bugtool.go (hunk 2 above)
Comment (tixxdz) [reply pkg/bugtool/bugtool.go:561]: good catch, will do ;-)

**Comment by jrfastab** [comment]
gops looks relatively small <10MB so should be ok.

**Pull request merged**